build/ublue-os-just/lib-ujust/ujust.sh

```diff
@@ -10,3 +10,5 @@ source /usr/lib/ujust/libfunctions.sh
 source /usr/lib/ujust/libdistrobox.sh
 # Import functionality related to toolbox
 source /usr/lib/ujust/libtoolbox.sh
+# Import functionality related to usage of run0
+source /usr/lib/ujust/librun0.sh
```

Comment:
I don't see `/usr/lib/ujust/librun0.sh` in existing images or even in this PR. Not sure how this was going to work, unless it was what got renamed to `librunasroot.sh`.

Reply:
Yes, I forgot to rename it, thanks for noticing

Reply:
Fixed